manifests/g/GDKSoftware/Duster/1.1.0.2/GDKSoftware.Duster.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GDKSoftware.Duster
+PackageVersion: 1.1.0.2
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://gdksoftware.com/Duster setup.exe
+  InstallerSha256: 8FDF36F38BD1369981EB9DD94C879E07B4E8A67E239D1CAB55403719CDC22940

```

Comment:
```suggestion
  InstallerSha256: 8FDF36F38BD1369981EB9DD94C879E07B4E8A67E239D1CAB55403719CDC22940
  ProductCode: Duster_is1
```

Reply:
because there's no `Publisher` for this application, we need `ProductCode` which is taken from the registry.